src/pages/guides/how-to-get-funding-for-your-open-source-projects.md. md

```diff
@@ -0,0 +1,51 @@
+# How To Get Funding For Your Open Source Projects
+
+
+# Overview:
```

Comment:
I'm not a super huge fan of having a h1 after another h1 for accessibility screenreader reasons, read [here](https://blog.shimin.io/should-you-use-multiple-h1-s-in-2022/#:~:text=Using%20more%20than%20one%20%3Ch1,is%20beneficial%20for%20screenreader%20users.)